bin/live/pycbc_live_supervise_single_significance_fits

```diff
@@ -389,28 +388,27 @@ def do_fitting(args, day_dt, day_str):
     combined_args = ['pycbc_live_combine_single_significance_fits']
     combined_args += dict_to_args(combined_options)
 
-    run_and_error(combined_args)
+    run_and_error(combined_args, controls)
 
-    logging.info('Copying combined fits file to local filesystem')
-    try:
-        shutil.copyfile(
-            combined_options['output'],
-            controls['variable-trigger-fits']
-        )
-    except Exception as e:
-        mail_volunteers_error(
-            controls,
-            [str(e)],
-            "PyCBC live could not copy to variable trigger fits file"
-        )
-        raise e
-
-
-    logging.info(
-        "%s updated to link to %s",
-        controls['variable-trigger-fits'],
-        combined_options['output']
-    )
+    if 'variable-trigger-fits' in controls:
+        logging.info('Copying combined fits file to local filesystem')
+        try:
+            shutil.copyfile(
+                combined_options['output'],
+                controls['variable-trigger-fits']
+            )
+            logging.info(
+                "%s updated to link to %s",
+                controls['variable-trigger-fits'],
+                combined_options['output']
+            )
```

Comment:
This can be left where it was; in case of an exception, it will not happen anyway.